packages/@react-spectrum/button/chromatic/Button.chromatic.tsx

```diff
@@ -12,16 +12,49 @@
 
 import Bell from '@spectrum-icons/workflow/Bell';
 import {Button} from '../';
+import {classNames} from '@react-spectrum/utils';
 import {Flex} from '@react-spectrum/layout';
+import {generatePowerset} from '@react-spectrum/story-utils';
+import {Grid, repeat} from '@react-spectrum/layout';
 import React from 'react';
 import {storiesOf} from '@storybook/react';
+import styles from '@adobe/spectrum-css-temp/components/button/vars.css';
 import {Text} from '@react-spectrum/text';
 
+let states = [
+  {UNSAFE_className: classNames(styles, 'is-hovered'), 'data-hover': true},
+  {UNSAFE_className: classNames(styles, 'is-active'), 'data-active': true},
+  {UNSAFE_className: classNames(styles, 'focus-ring'), 'data-focus': true},
+  {style: 'fill'},
+  {style: 'outline'},
+  {staticColor: 'white'},
+  {staticColor: 'black'},
+  {isDisabled: true}
+];
+
+let combinations = generatePowerset(states, v => (v.UNSAFE_className && v.isDisabled) || (v['data-focus'] && (v['data-hover'] || v['data-active'])) || (v['data-hover'] && v['data-active']));
```

Comment:
where do data-active/hover/focus get used in the css?

Reply:
They don't. They're just there so we can distinguish the states right here.